File: app/src/main/java/com/savvasdalkitsis/gameframe/injector/rx/RxSharedPreferencesInjector.java
package com.savvasdalkitsis.gameframe.injector.rx;

import com.savvasdalkitsis.gameframe.infra.preferences.RxSharedPreferences;

import static com.savvasdalkitsis.gameframe.injector.android.SharedPreferencesInjector.sharedPreferences;

public class RxSharedPreferencesInjector {

    public static RxSharedPreferences rxSharedPreferences() {
        return new RxSharedPreferences(sharedPreferences());
    }

}

File: app/src/main/java/com/savvasdalkitsis/gameframe/injector/rx/RxSharedPreferencesInjector.kt
package com.savvasdalkitsis.gameframe.injector.rx

import com.savvasdalkitsis.gameframe.infra.preferences.RxSharedPreferences

import com.savvasdalkitsis.gameframe.injector.android.SharedPreferencesInjector.sharedPreferences

object RxSharedPreferencesInjector {

    fun rxSharedPreferences() = RxSharedPreferences(sharedPreferences())

}
